cinema_service/settings.py

```diff
@@ -40,12 +40,14 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "rest_framework",
+    "debug_toolbar",
     "cinema",
     "user",
 ]
 
 MIDDLEWARE = [
     "django.middleware.security.SecurityMiddleware",
+    "debug_toolbar.middleware.DebugToolbarMiddleware",
```

Comment:
The 'DebugToolbarMiddleware' is included in MIDDLEWARE. Ensure it is removed or conditionally included for production environments.